Parsers/ASimWebSession/Test/Barracuda_WAF_ASimWebSession_DataTest.csv

```diff
@@ -0,0 +1,13 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 93 records (100.0%) for field [EventProduct] of type [Enumerated]: [""WAF""] (Schema:WebSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 93 records (100.0%) for field [EventVendor] of type [Enumerated]: [""Barracuda""] (Schema:WebSession)"
+"(1) Warning: Empty value in 8 records (8.6%)  in mandatory field [Dst] (Schema:WebSession)"
```

Comment:
Why are we getting empty string in this field?

Reply:
Fixed the issue and now we are not getting empty string.